frontend/app/professor/test-simulations/page.tsx

```
@@ -1417,309 +1817,603 @@ ${availablePersonas.map(persona => `• @${persona.name.toLowerCase().replace(/\
+                        marginTop: '0.5rem',
+                        fontSize: '0.95rem',
+                        whiteSpace: 'pre-wrap',
+                        width: '100%',
+                        fontFamily: 'inherit',
+                        resize: 'none',
+                        color: '#222'
+                      }}
+                      tabIndex={-1}
+                      aria-readonly="true"
+                    >
+                      {scene.user_responses && scene.user_responses.length > 0
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-readonly' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)